src/apps/vpn/models/servercity.cpp

```diff
@@ -108,15 +115,14 @@ const QString ServerCity::localizedName() const {
 }
 
 int ServerCity::connectionScore() const {
-  ServerCountryModel* scm = MozillaVPN::instance()->serverCountryModel();
+  ServerLatency* sl = MozillaVPN::instance()->serverLatency();
```

Comment:
Hope this isn't too nit-picky, but can we rename all `sl` def/declarations to `ServerLatency* serverLatency` for better readability?

Reply:
That is a good suggestion, thank you.